src/config.rs

```diff
@@ -221,7 +221,15 @@ pub(crate) struct AutolabelLabelConfig {
 #[derive(PartialEq, Eq, Debug, serde::Deserialize)]
 pub(crate) struct NotifyZulipConfig {
     #[serde(flatten)]
-    pub(crate) labels: HashMap<String, NotifyZulipLabelConfig>,
+    pub(crate) labels: HashMap<String, NotifyZulipNameConfig>,
+}
+
+#[derive(PartialEq, Eq, Debug, serde::Deserialize)]
+pub(crate) struct NotifyZulipNameConfig {
+    #[serde(flatten)]
+    pub(crate) default: Option<NotifyZulipLabelConfig>,
```

Comment:
@ismailarilik can you put a comment explaining what this field (is this a label?) is and why is it needed?

Reply:
Done.